pystac_monty/sources/usgs.py

```diff
@@ -52,7 +60,77 @@ def get_losses_data(self) -> list[dict[str, typing.Any]] | None:
         return self.losses_data or []
 
 
-class USGSTransformer(MontyDataTransformer[USGSDataSource]):
+class USGSSourceType(Enum):
+    EVENT = "geteventdata"
+    LOSS = "getlossdata"
+
+
+class USGSEvent(BaseModel):
+    type: str
+    data: GenericDataSource
+
+
+class USGSLossData(BaseModel):
+    type: str
+    data: GenericDataSource
+
+
+class USGSDataSourceV3(MontyDataSourceV3):
```

Comment:
Can we just use the name USGSDataSource instead of V3 ? Let's make it uniform across sources.